### DISCUSSION TITLE
The "setDefault" method does not work for forms of type "text" with name "value"

### DISCUSSION THREAD

**Discussion by hostbox2** [phalcon/cphalcon#16367] [Q&A]
The "setDefault" method does not work for forms of type "text" with name "value".

Example Form initialize:
class notificationsEditForm extends Form {
```php

        public function initialize($entity = null) {
...
            $value = new Text('value', array(
                'maxlength' => '20'
            ));
            $value->setDefault('10');
            $value->addValidators(array(new Between(array(
                'minimum' => -999999999,
                'maximum' => 999999999,
                'message' => 'The field is filled incorrectly',
            ))));
            $this->add($value);
...
        }
    }
``` 

Example Form in controller:
```php
$notificationsEditForm = new \dashboard\notificationsEditForm();
``` 

Example Form in view:
```php
<?=$notificationsEditForm->render('value')?>
``` 

As a result, the form is displayed, but it has no default value (just a blank field).

IF you rename the form, for example, call the field "valuer" - the default value is inserted

Phalcon 5.2.2

**Comment by s-ohnishi** [comment]
[issues](https://github.com/phalcon/cphalcon/issues/16382)
